### ISSUE TITLE
Link role-dependencies to github repo

### ISSUE THREAD

**Issue by ypid** [nickjj/ansigenome#14]
Example: https://github.com/debops/ansible-apt#role-dependencies

`debops.etc_services` could link to https://github.com/debops/ansible-etc_services

**Comment by drybjed** [comment]
That could probably be done in Jinja template itself with some text transformation... I'll look if that's possible.

**Comment by nickjj** [comment]
That is true. Let me know how it goes drybjed. I feel like this is a template concern. If it's reasonable it could be added to the default template.
